tests/db/model/bot/utils.js

```diff
@@ -21,6 +21,12 @@ const standard = {
   name: n,
   url: 'http://www.bar.com',
   active: true,
+  settings: [
+    {
+      key: 'key1',
+      helpText: 'helpText1'
```

Comment:
Can you add a test for: "This is some text that helps to understand what the settings should do"